releasenotes/notes/0.15/add-rep-delay-c97d5aa8fc9696da.yaml

```diff
@@ -26,6 +26,6 @@ features:
     property for pulse qobjs.
   - |
     The ``backend_configuration_schema.json`` JSON Schema file in
-    mod:`qiskit.schemas` has been updated to include ``rep_delay_range`` and
+    :mod:`qiskit.schemas` has been updated to include ``rep_delay_range`` and
```

Comment:
Thanks for updating this here, these reno yaml files are the source of truth for release notes. But for the time being we also have to update these in https://github.com/Qiskit/qiskit/blob/master/docs/release_notes.rst for things to be reflected in the documentation. Hopefully in the near future this duplicate copy won't need to exist, I just need to find time to implement: https://github.com/Qiskit/qiskit/issues/997 Once that's in place just updating the per repo copy is sufficient.

Reply:
Are you asking me to update ``qiskit/qiskit/docs/release_notes.rst``, or will you push that after this?

Reply:
I was asking if you would push a follow up PR to make the changes there in `qiskit/qiskit`. But if you'd prefer I can do it too.

Reply:
I can do that once this is merged 👍